src/Page/Story/View.elm

```diff
@@ -116,3 +116,10 @@ imageCaptionStyle : Style
 imageCaptionStyle =
     batch
         [ margin3 (rem 0) (rem 0) (rem 1) ]
+
+
+imageSizeFixStyle : Style
+imageSizeFixStyle =
```

Comment:
Is there are reason why this width couldn't be added in to the existing featureImageStyle? I think the feature image is only used in one other place, and is it possible that the same thing could be happening there that is happening here? 

I ask because having separate styles that you add one at a time like this can get really confusing unless they're absolutely necessary to change the behaviour of the original css. Also the name of this is more like a PR comment than a reference to what the styles are actually doing - shouldn't it be called something like image100PctWidth if this needs to be kept as a separate style?

Reply:
You make a good point.

I have moved the rule into the `featureImageStyle` which is used in show Guide and show Story views.

My first solution was trying to be conservative and only fixing the one problem in isolation but i suppose such an approach could easily make spaghetti css.